app/ui/src/app/support/support.component.html

```diff
@@ -0,0 +1,77 @@
+<div class="row">
+  <div class="col-md-12">
+    <h1>Support</h1>
+    <p>To obtain support, download diagnostic information through this page. </p>
+    <p>If the downloaded zip file is less than 1/2 GB then attach it to an email message that requests help and send the message to <a href="mailto:fuse-online-tech-preview@redhat.com">fuse-online-tech-preview@redhat.com</a>. </p>
+      <p>If the downloaded zip file is larger then 1/2 GB, send an email message to request help, do not attach the file but indicate that the zip file is too large to attach. 
+        Technical support will provide instructions for sending the zip file.</p>
+
+    <h1>Version</h1>
+    <p class="version"><strong>Syndesis</strong> v1.0</p>
+
+    <h1>Download Trobuleshooting Diagnostics</h1>
+    <p>
+        System level and application level diagnostics will be captured since both are required to troubleshoot any issues. Usernames and passwords are not present in the downloaded zip file. 
+    </p>
+
+    <div class="mainform">
+      <p>Select the integrations to capture diagnostics for:</p>
+      <form novalidate>
+        <fieldset>
+            <p> 
+              <input type="radio" id="alllogs" name="logs" [value]="true" [(ngModel)]="allLogsSelected" (click)="deselectAll()">
```

Comment:
I don't think you want to combine `ngModel` and a `click` handler for controlling the list selection, that'll potentially trigger multiple changes.

Reply:
Thanks. I have removed references to `ngModel` and also to `@ViewChild` usage that were a leftover of an older spike.